portal/server/index.js

```diff
@@ -94,6 +102,15 @@ app.use(
 
 app.get('*', (req, res) => res.render('page-not-found.njk', { user: req.session.user }));
 
-console.info(`GITHUB_SHA: ${process.env.GITHUB_SHA}`);
+// error handler
+app.use((err, req, res, next) => {
+  if (err.code === 'EBADCSRFTOKEN') {
+    // handle CSRF token errors here
+    res.status(err.statusCode || 500);
+    res.redirect('/');
```

Comment:
Should this redirection be an error page rather than home page? Thinking of GDS pattern

Reply:
I was discussing this the other day with @abhi-markan . We agreed to redirect to the main page for security reasons :)